src/en/news/blog/2025/v19-2-1-squid-released/index.md

```diff
@@ -0,0 +1,700 @@
+
+- squid: rgw: optimize bucket listing to skip past regions of namespaced entries ([pr#61070](https://github.com/ceph/ceph/pull/61070), J. Eric Ivancich)
+
+- squid: rgw: revert account-related changes to get\_iam\_policy\_from\_attr() ([pr#59221](https://github.com/ceph/ceph/pull/59221), Casey Bodley)
+
+- squid: rgw: RGWAccessKey::decode\_json() preserves default value of 'active' ([pr#60823](https://github.com/ceph/ceph/pull/60823), Casey Bodley)
+
+- squid: rgw: switch back to boost::asio for spawn() and yield\_context ([pr#60133](https://github.com/ceph/ceph/pull/60133), Casey Bodley)
+
+- squid: rgwlc: fix typo in getlc (ObjectSizeGreaterThan) ([pr#59223](https://github.com/ceph/ceph/pull/59223), Matt Benjamin)
+
+- squid: RGW|BN: fix lifecycle test issue ([pr#59010](https://github.com/ceph/ceph/pull/59010), Ali Masarwa)
+
+- squid: RGW|Bucket notification: fix for v2 topics rgw-admin list operation ([pr#60774](https://github.com/ceph/ceph/pull/60774), Oshrey Avraham, Ali Masarwa)
+
+- squid: seastar: update submodule ([pr#58955](https://github.com/ceph/ceph/pull/58955), Matan Breizman)
+
+- squid: src/ceph\_release, doc: mark squid stable ([pr#59537](https://github.com/ceph/ceph/pull/59537), Neha Ojha)
+
+- squid: src/crimson/osd/scrub: fix the null pointer error ([pr#58885](https://github.com/ceph/ceph/pull/58885), junxiang Mu)
+
+- squid: src/mon/ConnectionTracker<span></span>.cc: Fix dump function ([pr#60003](https://github.com/ceph/ceph/pull/60003), Kamoltat)
+
+- squid: suites/upgrade/quincy-x: update the ignore list ([pr#59624](https://github.com/ceph/ceph/pull/59624), Nitzan Mordechai)
+
+- squid: suites: adding ignore list for stray daemon ([pr#58267](https://github.com/ceph/ceph/pull/58267), Nitzan Mordechai)
+
+- squid: suites: test should ignore osd\_down warnings ([pr#59147](https://github.com/ceph/ceph/pull/59147), Nitzan Mordechai)
+
+- squid: test/neorados: remove depreciated RemoteReads cls test ([pr#58144](https://github.com/ceph/ceph/pull/58144), Laura Flores)
+
+- squid: test/rgw/notification: fixing backport issues in the tests ([pr#60545](https://github.com/ceph/ceph/pull/60545), Yuval Lifshitz)
+
+- squid: test/rgw/notification: use real ip address instead of localhost ([pr#59303](https://github.com/ceph/ceph/pull/59303), Yuval Lifshitz)
+
+- squid: test/rgw/notifications: don't check for full queue if topics expired ([pr#59917](https://github.com/ceph/ceph/pull/59917), Yuval Lifshitz)
+
+- squid: test/rgw/notifications: fix test regression ([pr#61119](https://github.com/ceph/ceph/pull/61119), Yuval Lifshitz)
+
+- squid: Test: osd-recovery-space<span></span>.sh extends the wait time for "recovery toofull" ([pr#59041](https://github.com/ceph/ceph/pull/59041), Nitzan Mordechai)
+
+- upgrade/cephfs/mds\_upgrade\_sequence: ignore osds down ([pr#59865](https://github.com/ceph/ceph/pull/59865), Kamoltat Sirivadhna)
```

Comment:
Is this missing "squid" or does it not need it because it applies to multiple major releases?

Reply:
must be from the PR commit comment